pkgs/anordal/shellharden/registry.yaml

```diff
@@ -0,0 +1,33 @@
+packages:
+  - type: github_release
+    repo_owner: anordal
+    repo_name: shellharden
+    aliases:
+      # The registry had registered this name when this tool was first registered.
+      # https://github.com/aquaproj/aqua-registry/pull/13784
+      - name: crates.io/shellharden
+    description: The corrective bash syntax highlighter
+    version_constraint: "false"
+    version_overrides:
+      - version_constraint: semver("<= 4.3.0")
+        type: cargo
+        crate: shellharden
```

Comment:
I wrote this code with reference to [#17534](https://github.com/aquaproj/aqua-registry/pull/17534/commits/1004bddb5ecb8323a45941ca4083f61e20fcfc25).